### PR TITLE
Submitting Party on the Review screen

### PR DESCRIPTION
**Pull request by dimak1**
*Issue #:* /bcgov/entity#13823

*Description of changes:*
- Add Submitting Party on the Review page via the new CurrentUser common component
- Pull Account name, Mailing address, Account admin's email and Account admin's phone number from `/orgs` Auth API
- Add unit tests

Screen Shot 2022-10-26 at 10 02 53 AM

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the PPR license (Apache 2.0).

### CODE REVIEW

File: ppr-ui/src/components/common/AccountInfo.vue [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+              <td class="py-6">
+                {{ accountInfo.accountAdmin.email }}
+              </td>
+              <td class="py-6">
+                {{ toDisplayPhone(accountInfo.accountAdmin.phone) }}
+                <span v-if="accountInfo.accountAdmin.phoneExtension">
+                  Ext {{ accountInfo.accountAdmin.phoneExtension }}
+                </span>
+              </td>
+            </tr>
```
Comment (cameron-eyds) [inline ppr-ui/src/components/common/AccountInfo.vue:66]: Not a huge deal but there is a lot of tags here that could very easily fit on 1 line. 
I guess it's preference but the condensing might make for better readability. Your call though. 
ie `<th class="pl-6 py-4">Name</th>` as example, i'm not sure its easier to read as 3 lines vs 1.

File: ppr-ui/src/utils/auth-helper.ts [hunk 2]
```diff
@@ -64,6 +65,73 @@ export async function getStaffegisteringParty (isBcOnline: boolean): Promise<Par
...
+        region: data?.mailingAddress?.region || '',
+        postalCode: data?.mailingAddress?.postalCode || '',
+        country: data?.mailingAddress?.country || ''
+      }
+
+      return {
+        id: data.id,
+        isBusinessAccount: data.isBusinessAccount,
+        name: data.businessName || data.name,
+        mailingAddress: address
```
Comment (cameron-eyds) [inline ppr-ui/src/utils/auth-helper.ts]: Does address need to be parsed out here? Would `mailingAddress: data.mailingAddress` cause an issue? 
Or is it that there is potential `null` values and you wanted string fallbacks?

**Review by cameron-eyds — APPROVED** [review]
Looks good, just super minor comments 👍

File: ppr-ui/src/utils/auth-helper.ts (hunk 2 above)
Comment (dimak1) [reply ppr-ui/src/utils/auth-helper.ts]: Good question. I was parsing this the same way it is currently done for another GET call to **/orgs**. But it is true, I might not need to parse here. I'll update it. Good catch!

Comment (dimak1) [reply ppr-ui/src/utils/auth-helper.ts]: @cameron-freshworks updated

File: ppr-ui/src/components/common/AccountInfo.vue (hunk 1 above)
Comment (dimak1) [reply ppr-ui/src/components/common/AccountInfo.vue:66]: Formatted by Prettier 🤷🏻 I want to review its config once I get to it sometime later.

Comment (cameron-eyds) [reply ppr-ui/src/components/common/AccountInfo.vue:66]: haha all good. Like i said, this is just preference. 
Up to you!

**Review by ChasBoucher — APPROVED** [review]
Looks good

**Pull request merged**